### ISSUE TITLE
Leading space before an inner tag should be ignored.

### ISSUE THREAD

**Issue by yjukaku** [yjukaku/html2lucky#3]
Hello, and thanks for your work on this project.

I have a found a bug when using this site. Here are the details:

### The minimal amount of HTML input that triggers the bug

```html
<div> <a>Hello</a> </div>
```

### What I expected to be output when inputting the minimal HTML above

```crystal
div do
  a "Hello"
end
```

### What is actually output by the site

```crystal
div do
  text " "
  a "Hello"
  text " "
end
```

I will wait patiently as you work on this free project in your precious spare time. Thank you again!

**Comment by yjukaku** [comment]
I'm actually not sure if this is correct behavior or not. Need some expert eyes on this.

**Comment by yjukaku** [comment]
This has been closed by 3fb2a2d
